### ISSUE TITLE
Does it still support `Set("title = ?title")` this way, or bug?

### ISSUE THREAD

**Issue by danclive** [uptrace/bun#109]
I see on the [doc](https://bun.uptrace.dev/guide/queries.html#update):

```go
res, err := db.NewUpdate().Model(book).Set("title = ?title").WherePK().Exec(ctx)
```

When I use it this way:

```go
db.NewUpdate().Model(&device).Set("link = ?link").WherePK().Exec(ctx)
```

But it prints an error:

```
[bun]  13:35:29.100   UPDATE  10.386ms  UPDATE "device" AS "device" SET link = ?link WHERE "deleted" IS NULL AND ("id" = '017b5480ec95d02c88fc3002') 	  pgdriver.Error: ERROR #42883 operator does not exist: ? integer 
```

Is this a bug or is bun no longer supported and the documentation is just not up to date?

**Comment by vmihailenco** [comment]
It probably should be supported to ease migration. I will take a look.

**Comment by vmihailenco** [comment]
Should be fixed in v0.4.3
